ui/src/app/base/components/NotificationGroup/Notification/__snapshots__/Notification.test.tsx.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`NotificationGroupNotification renders 1`] = `
 <NotificationGroupNotification
-  id={1}
+  id={7}
```

Comment:
Is this snapshotting a random int?

Reply:
I think it's semi-random. When you re-run the tests they pass. I think the changes I made to set up other models in the beforeEach meant that the random numbers where recalculated.

Reply:
If I move the `renders` test to be second in the list of `NotificationGroupNotification` tests, it fails. I think what's happening is that now the test success is determined by the order the tests are run, because `id` is sequential. Giving the notification a defined id should solve the problem. Or you could remove the test, not sure how useful the snapshot test is given how many unit tests there are.

Reply:
I've set a fixed `id` for this now. Thanks.